test/C/src/concurrent/AsyncCallbackDrop.lf

```diff
@@ -35,7 +35,7 @@ main reactor {
 	state expected_time:time(100 msec);
 	state toggle:bool(false);
 
-    physical action a(100 msec, 100 msec, 'drop'):int;
+    physical action a(100 msec, 100 msec, "drop"):int;
```

Comment:
This makes me wonder which consequence the grammar change has in other places. Would `state foo:char('f')` be legal LF code? Or would we need to write `state foo:char({='f'=})`?

Reply:
(Since my changes, this comment is outdated, `state foo:char('f')` is supported and tested.)